### DISCUSSION TITLE
Question - Purpose of simulation_frequency

### DISCUSSION THREAD

**Discussion by mhtb32** [eleurent/highway-env#175] [General]
I have two questions about the code:

1. I think that the purpose behind `simulation_frequency` config is to make sure simulation time difference is always kept constant at the desired value, is that right?
2. If one wants to give the agent a new input value at every step, Does he/she should use a `policy_frequency` the same as the `simulation_frequency`?

**Comment by eleurent** [comment]
1. Yes. Since Euler integration is used, higher frequency is more accurate but also more demanding computationally.
2. Absolutely, typically for a continuous control task. Conversely, it does not make much sense for temporally extended actions like lane changes maneuvers.
